src/azure-cli/azure/cli/command_modules/marketplaceordering/_help.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=wildcard-import
+# pylint: disable=unused-wildcard-import
+from knack.help_files import helps  # pylint: disable=unused-import
+
+from .generated._help import helps
+try:
+    from .manual._help import helps  # pylint: disable=reimported
+except ImportError:
+    pass
```

Comment:
This file should be generated by codegen. It seems that your generated file is different than mine.
```
from .generated._help import helps  # pylint: disable=reimported
try:
    from .manual._help import helps  # pylint: disable=reimported
except ImportError as e:
    if e.name.endswith('manual._help'):
        pass
    else:
        raise e
```